doc/rados/operations/balancer.rst

```diff
@@ -19,17 +19,17 @@ The current status of the balancer can be checked at any time with::
 Automatic balancing
 -------------------
 
-The automatic balancing can be enabled, using the default settings, with::
-
-  ceph balancer on
-
-The balancer can be turned back off again with::
+The automatic balancing feature is enabled by default in the ``upmap``
+mode. The balancer can be turned back off with::
 
   ceph balancer off
 
-This will use the ``crush-compat`` mode, which is backward compatible
-with older clients, and will make small changes to the data
-distribution over time to ensure that OSDs are equally utilized.
+The ``upmap`` mode needs ``require_min_compat_client`` luminous, which
```

Comment:
you can reference rados/operations/upmap.rst instead - the underscores vs dashes in the user command may confuse users